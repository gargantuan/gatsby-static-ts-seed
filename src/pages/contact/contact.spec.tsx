import * as React from 'react';
import * as renderer from "react-test-renderer"
import Contact from './index';

describe("Contact", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Contact />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
