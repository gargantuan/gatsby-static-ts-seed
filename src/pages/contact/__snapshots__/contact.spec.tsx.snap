// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact renders correctly 1`] = `
Array [
  <h1>
    <a
      href="/"
    >
      Website Title
    </a>
  </h1>,
  <nav>
    <ul>
      <li>
        <a
          href="/about"
        >
          About
        </a>
      </li>
      <li>
        <a
          href="/contact"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>,
  <h1>
    Contact
  </h1>,
  <p>
    Contact goes here...
  </p>,
  <footer>
    <p>
      <a
        href="/privacy"
        title="privacy policy"
      >
        Privacy policy
      </a>
    </p>
  </footer>,
]
`;
